src/main.go

```diff
@@ -267,22 +273,27 @@ func runCommand(name string, args ...string) (stdout string, stderr string, exit
 	return
 }
 
-func parseOutput(output string) (string, string, map[string]float64) {
+func parseOutput(output string) (map[string]string, error) {
 	match := reOutput.FindStringSubmatch(output)
 	result := make(map[string]string)
-	for i, name := range reOutput.SubexpNames() {
+	subexpNames := reOutput.SubexpNames()
+	if len(subexpNames) > len(match) {
+		return nil, errLessMatchesFound
+	}
+	if len(subexpNames) < len(match) {
+		log.Warn("More matches found than expected, command output -> ", output)
+	}
```

Comment:
I think that this is not actually possible 🤔 
I was reading the docs and I do not think it can happen